app/views/projects/new.html.slim

```diff
@@ -31,11 +31,12 @@ p.well Please make sure you have read our #{link_to "mentor guide", 'http://rail
       = f.input :source_url, as: :url, wrapper_html: { class: 'col-md-6' }, label: "Repository", placeholder: 'Link to your source code (Github, Gitlab, Bitbucket, …)' 
       = f.input :description, wrapper_html: { class: 'col-md-6' }, placeholder: "What's your project about?", input_html: { rows: 5 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
       = f.input :issues_and_features, wrapper_html: { class: 'col-md-6' }, placeholder: "A list of possible features to work on, maybe even concrete Github issues", input_html: { rows: 5 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
```

Comment:
I suggest to wide the scope of this question - by making the hint more general 😉 
How about:
> How will the students be involved? What will they be working on? Possible features, maybe links to concrete issues on Github.

I liked how maintainers came up with special *"projects"* within their projects for RGSoC including guidance, etc. that's why I think *"How will the students be involved"* could also help here to a) get more meaningful answers and b) have the mentors think about this in advance.